examples/statistics/boxplot_color_demo.py

```diff
@@ -1,4 +1,4 @@
-# Box plots with custom fill colors
+"""Box plots with custom fill colors."""
```

Comment:
You are making me work :)

Something like: 
"This plot illustrates how to create two types of boxplots (rectangular and notch), and how to fill them with custom colors by accessing some properties of the artists of the boxplots."
If we'd go with this description, I'd replace the title with something shorter: "Colored box plots"

But with this description, I'd remove the horizontal grid lines and replace the `setp` with `ax.set_xticklabels`.